collie/model/base/base_pipeline.py

```diff
@@ -210,6 +210,13 @@ def __init__(self,
             self.hparams.num_items = self.train_loader.num_items
             self.hparams.num_epochs_completed = 0
 
+            # check there are no nulls in item_metadata
```

Comment:
[Optional, slight language tweak for clarity] 

```suggestion
            # ensure there are no nulls in ``item_metadata``
```

Reply:
done